backend/README.md

```diff
@@ -79,9 +79,9 @@ More details about our GraphQL playground and how to use it with ocelot.social c
 
 ![GraphQL Playground](../.gitbook/assets/graphql-playground.png)
 
-### Database Indices and Constraints
+### Database Indexes and Constraints
```

Comment:
indices is correct isn't it?

Reply:
Indexes is correct says Neo4j, see headline:
https://neo4j.com/docs/cypher-manual/current/indexes-for-search-performance/